### PR TITLE
Mariana soltuz

### PR DESCRIPTION
**Pull request by marianasoltuz**
For Practice

### CODE REVIEW

**Review by andrewbalint51 — CHANGES_REQUESTED** [review]
Looks like pull request was submitted correctly! Just need to complete the code before the merge is finalized. It only shows me empty classes right now. Is this right?

**Comment by AndrewSloth** [comment]
> Looks like pull request was submitted correctly! Just need to complete the code before the merge is finalized. It only shows me empty classes right now. Is this right?

I will dismiss your request for change just for her practice. So she will be available to try merging function

**Review by AndrewSloth — APPROVED** [review]

**Pull request merged**
